content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/100-cyclic-referential-actions.mdx

```diff
@@ -11,61 +11,66 @@ Circular references can occur between models when each model references another,
 
 With Microsoft SQL Server however, if the [referential actions](./) on the relation field are set to something other than `NoAction`, the data model validator will throw an error about the cyclic relation.
```

Comment:
```suggestion
With Microsoft SQL Server however, if the [referential actions](./) on the foreign key are set to something other than `NO ACTION`, it will check for cycles or multiple cascade paths and return an error when executing the SQL.
```